src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1,5 +1,6 @@
-
 """
+    AbstractDataFrame
+
```

Comment:
...interface for representing data in relational format (or column-oriented data).

Database-like seems too broad.